components/root_layout/root_layout_default.story.vue

```diff
@@ -25,7 +26,236 @@
       <span v-html="sidebar" />
     </template>
     <template v-if="defaultSlot">
-      <span v-html="defaultSlot" />
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam dignissim eleifend condimentum. Vestibulum
```

Comment:
I think this can be removed from the template by repeating a sample text passed from the story definition like the pattern [we've followed in the modal](https://github.com/dialpad/dialtone-vue/blob/staging/components/modal/modal.stories.js#L148).

Also make the sample text fallbacks the defaultSlot.
```html
<template>
    <span v-if="defaultSlot" v-html="defaultSlot" />
    <span v-else>{{ sampleText}}</span>
</template>

```

Reply:
well that's much cleaner, thanks.